solr/core/src/java/org/apache/solr/spelling/WordBreakSolrSpellChecker.java

```diff
@@ -62,15 +62,15 @@ public class WordBreakSolrSpellChecker extends SolrSpellChecker {
   public static final String PARAM_MIN_BREAK_WORD_LENGTH = "minBreakLength";
 
   /** See {@link BreakSuggestionTieBreaker} for options. */
-  public static final String PARAM_BREAK_SUGGESTION_TIE_BREAKER = "breakSugestionTieBreaker";
+  public static final String PARAM_BREAK_SUGGESTION_TIE_BREAKER = "breakSuggestionTieBreaker";
```

Comment:
subjective: Wondering if for the 9.x line of releases we might want to maintain backwards compatibility for user convenience e.g.
```suggestion
  @Deprecated since 9.6 and will be removed from 10.0
  private static final String PARAM_BREAK_SUGESTION_TIE_BREAKER = "breakSugestionTieBreaker";

  public static final String PARAM_BREAK_SUGGESTION_TIE_BREAKER = "breakSuggestionTieBreaker";
```

here and reading both in the code. That way users upgrading (say) from 9.5 to 9.6 need not upgrade their configuration just as yet but they would receive a friendly WARNing logging or so to let them know about the rename.

Reply:
Good catch...      In your suggested commit you don't have the warning right?  Okay, I guess what the plan is is to add the warning for the deprecation..   commit this and backport, and then in main we remove the mispelled variable.     and mention that "breaking change" in the upgrade notes to 10?

Reply:
Thanks for the inputs @cpoerschke and @epugh - I added the deprecation warning to the logs, and tweaked the changes entry to mention that the old parameter is deprecated.